lib/render-footer.js

```diff
@@ -11,61 +11,56 @@ export const renderFooter = ({
 	csvFilename,
 	xlsxFilename,
 	xlsxSheetname,
-	topPlacement
-}) => {
-	return html`
-		<cosmoz-bottom-bar
-			id="bottomBar"
-			?active=${!isEmpty(selectedItems.length)}
-		>
-			<slot name="info" slot="info">
-				${ngettext(
-					'{0} selected item',
-					'{0} selected items',
-					selectedItems.length
-				)}
-			</slot>
-			<slot name="actions" id="actions"></slot>
-			<!-- These slots are needed by cosmoz-bottom-bar
+	topPlacement,
+	maxToolbarItems,
+}) =>
+	html`<cosmoz-bottom-bar
+		id="bottomBar"
+		?active=${!isEmpty(selectedItems.length)}
+		?max-toolbar-items=${!isEmpty(maxToolbarItems)}
```

Comment:
`?prop` should only be used for boolean attributes, for number props you should use `.prop`. 

```suggestion
		.maxToolbarItems=${maxToolbarItems}
```

Reply:
Keeping this change after our discussion in slack 🙂